docs/sql/index.md

```diff
@@ -91,3 +93,56 @@ POST _opendistro/_sql/_explain
   "size": 50
 }
 ```
+
+### Syntax analysis exception
+
+
+If you get the following "Invalid SQL query" error:
```

Comment:
You might receive the following error if the plugin can't parse your query:

Reply:
Done.